recipes/deepfplearn/meta.yaml

```
@@ -30,6 +34,10 @@ requirements:
     - tensorflow-base
     # - tensorflow-gpu =2
     - wandb =0.12
+    - chemprop
+    - seaborn =0.12.2
+    - umap
+    - argparse
```

```suggestion
    - umap-learn =0.5.3
```

@soulios is this correct (if so we also need to update it in our `setup.py`)